packages/jaeger-ui/src/components/DeepDependencies/Header/index.css

```diff
@@ -43,3 +45,51 @@ limitations under the License.
   padding: 0.75rem 1.25rem 0.75rem 1.25rem;
   position: relative;
 }
+
+.DdgHeader--uiFind {
+  align-items: center;
+  background-color: #ffffff;
+  border-left: 1px solid #eee;
+  border-right: 1px solid #eee;
+  color: #bbb;
+  display: flex;
+  flex-grow: 1;
+  margin-right: 8px;
+  max-width: 20%;
+  min-width: 200px;
+  transition: max-width 0.5s;
+}
+
+.DdgHeader--uiFind:focus-within {
+  max-width: 60%;
```

Comment:
550px seems pretty small. it would go from 20% to ~28% at 1080p, or 20% to ~21.4% at 1440p, or 20% to ~16% on an ultrawide (as in it actually shrinks!)
I updated to 40% so that it doubles in width instead of tripling.

Reply:
550px is pretty wide for an input. For instance, in Chrome the find input is around 210px. 

Here is the 40% on my laptop:

<img width="1880" alt="Screen Shot 2019-08-03 at 11 44 24 PM" src="https://user-images.githubusercontent.com/2304337/62419206-db9e5c80-b648-11e9-8d3c-eb0c036a7e30.png">

I would say that's pretty huge. Doubling in size is a lot of movement, which is a jarring aesthetic. But, slowing down the animation makes the UI feel lethargic.

Reply:
Updated for the `max-width` to change between `400px` and `550px`